pyproject.toml

```diff
@@ -3,7 +3,7 @@ name = "blackbox-cli"
 packages = [
     { include = "blackbox" }
 ]
-version = "v2.3.2"
+version = "v2.4.0"
```

Comment:
Don't increment the version number manually, this is handled automatically by CI/CD, see https://github.com/lemonsaurus/blackbox/blob/main/.github/workflows/publish.yaml

Reply:
I've rebased all the commits interactively to keep the history clean, so this bump is no longer part of d434891e80840ea61e1a3e08ead42886a4affd8f